apps/marketplace/components/BuyerDashboard/BuyerDashboard.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable */
+import React from 'react'
+import styles from './BuyerDashboard.scss'
+
+const BuyerDashboard = props => {
+  return (
+    <div className={styles.container}>
+      <div className={`${styles.header} row`}>
+        <div className="col-md-12 col-sm-12">
+          <span className="uikit-display-5">Opportunities</span>
```

Comment:
on mobile (chrome) this title sits over the buttons

Reply:
Yes, we don't have mobile version yet of buyer dashboard. Ingrid has to do one.